br/pkg/lightning/restore/restore.go

```diff
@@ -1349,6 +1349,9 @@ func (rc *Controller) restoreTables(ctx context.Context) error {
 			if err != nil {
 				return errors.Trace(err)
 			}
+			if cp.Status < checkpoints.CheckpointStatusAllWritten && len(tableMeta.DataFiles) == 0 {
```

Comment:
Why `cp.Status < checkpoints.CheckpointStatusAllWritten` is needed here?

Reply:
If `cp.Status >= CheckpointStatusAllWritten` then the restore process has no relation with the source files anymore, so we may allow it to continue even if we find there are no source files. Maybe there are situations that the source files are corrupted, the import phase can still go on.😅

Reply:
Comparing statue is wired. Status is not a clear name, Rename it to stage or something similar would be more clear.